python/nav/models/manage.py

```diff
@@ -2365,6 +2372,13 @@ class Sensor(models.Model):
     alert_type = models.IntegerField(
         db_column='alert_type', choices=ALERT_TYPE_CHOICES, null=True
     )
+    threshold_type = models.IntegerField(
+        db_column='threshold_type', choices=THRESHOLD_TYPE_CHOICES, null=True
+    )
+    threshold_alert_type = models.IntegerField(
+        db_column='threshold_alert_type', choices=ALERT_TYPE_CHOICES, null=True
```

Comment:
Reuses pre-existing alert types. Figured this could give some consistency, as the already existing "Warning" and "Alert" match quite well with the "Alarm" and "Warning" type thresholds.